public/static/locales/es/common.json

```
@@ -175,6 +175,7 @@
   "welcome_to_calcom": "Bienvenido a Cal.com",
   "welcome_instructions": "Cuéntanos cómo llamarte y dinos la zona horaria en la que estás. Podrás editarla más tarde.",
   "connect_caldav": "Conectar con el servidor de CalDav",
+  "connect_exchnage": "Conectar con el servidor de Exchange",
```

```suggestion
  "connect_exchange": "Conectar con el servidor de Exchange",
```
Exchange is misspelled here :)